ce/generators/msbuild-generator.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+const XMLWriter = require('xml-writer');
+
+export function generate_msbuild(locations: Array<[string, string]>, properties: Array<[string, Array<string>]>) : string {
```

Comment:
Will this allow PATH like locations (i.e. ';' separated lists of locations when generated) which multiple artifacts can add to?

Reply:
At the moment I believe "locations" is "the location of this thing" which doesn't make sense to have multiple answers. But no reason such a thing can't be added.